docs/guides/qiskit-code-assistant-local.mdx

```diff
@@ -0,0 +1,201 @@
+
+# Use Qiskit Code Assistant in local mode
+
+Learn how to install, configure, and use the Qiskit Code Assistant model on your local machine.
+
+<Admonition type="note" title="Notes">
+    - Qiskit Code Assistant is in preview release status and is subject to change.
+    - If you have feedback or want to contact the developer team, use the [Qiskit Slack Workspace channel](https://qiskit.enterprise.slack.com/archives/C07LYA6PL83) or the related public GitHub repositories.
+</Admonition>
+
+## Download the Qiskit Code Assistant model
+
+The Qiskit Code Assistant model is available in  <DefinitionTooltip definition="GGUF is a binary format that is designed for fast loading and saving of models, and for ease of reading.">GGUF file format</DefinitionTooltip> and can be downloaded from the Hugging Face Hub in one of two ways.
+
+<details>
+
+<summary>Download from the Hugging Face website</summary>
+
+Follow these steps to download the Qiskit Code Assistant GGUF model from the Hugging Face website:
+
+1. Navigate to the [`granite-8b-qiskit`](https://huggingface.co/qiskit) model page
+1. Select the `granite-8b-qiskit` GGUF model version
+1. Go to the Files and Versions tab and download the GGUF model
+
+</details>
+
+
+<details>
+
+<summary>Download using the Hugging Face CLI</summary>
+
+To download the `granite-8b-qiskit` GGUF model using the Hugging Face CLI, follow these steps:
+
+1. Install the [Hugging Face CLI](https://huggingface.co/docs/huggingface_hub/main/en/guides/cli)
+1. Log in to your Hugging Face account
+
+    ```
+    huggingface-cli login
+    ```
+
+1. Download the `granite-8b-qiskit` GGUF model
+
+    ```
+    huggingface-cli download <HF REPO NAME> <GGUF PATH> --local-dir <LOCAL PATH>
+    ```
+
+</details>
+
+
+## Use the Qiskit Code Assistant model
+
+There are multiple ways to deploy and interact with the downloaded `granite-8b-qiskit` GGUF model. This guide demonstrates using [Ollama](https://ollama.com) as follows: either with the [Ollama application](#using-the-ollama-application), or with the [`llama-cpp-python` package](#using-the-llama-cpp-python-package).
+
+### Using the Ollama application
+
+The Ollama application provides a simple solution to run the GGUF models locally. It is easy to use, with a CLI that makes the whole setup process, model management, and interaction fairly straightforward. It’s ideal for quick experimentation and for users that want fewer technical details to handle.
+
+#### Install Ollama
+
+1. Download the [Ollama application](https://ollama.com/download)
+1. Install the downloaded file
+1. Launch the installed Ollama application
+
+    <Admonition type="info">The application is running successfully when the Ollama icon appears in the desktop menu bar. You can also verify the service is running by going to http://localhost:11434/.</Admonition>
+
+1. Try Ollama in your terminal and start running models. For example:
+
+    ```
+    ollama run granite3-dense:8b
+    ```
+
```

Comment:
#### Using the Ollama/Hugging Face Hub integration

The [Ollama/Hugging Face Hub integration](https://huggingface.co/docs/hub/ollama) provides a way to interact with GGUF models hosted on the Hub without creating a new ```modelfile```.
1. Enable ollama under your [Local Apps settings](https://huggingface.co/settings/local-apps)
2. Go the [`granite-8b-qiskit`](https://huggingface.co/qiskit) GGUF model page, choose ollama from the ```Use this model``` dropdown. 
3. Run ```ollama run hf.co/Qiskit/granite-8b-qiskit-GGUF``` on the terminal

 A default ```template``` and ```params``` files are available by default on Hugging Face together with the GGUF model.

Reply:
thanks @lucaburatti7
https://github.com/Qiskit/documentation/pull/2275/commits/96a03ac1c945e8e3275f1743ee76117f8dbeb04d